hw/hw03/oyakovenko/hw3_1.py

```diff
@@ -0,0 +1,21 @@
+#open file with text (file should exists in the same directory with the current script)
+f = open("Zen.txt", "r")
+f_string = f.read()
+
+#find the number of occurrences of the words and print
+find_better = f_string.find('better')
+find_never = f_string.find('never')
+find_is = f_string.find('is')
```

Comment:
should have used "count"
since the search returns the index of occurrence and not the count